README.md

```diff
@@ -31,6 +31,24 @@ Also:
 
 ## Anvil
 
+### Dependencies
+You'll need to install the following dependencies; note, that those package
+names may vary depending on your OS and linux distribution:
+- `libdbus-glib-1`
+- `libudev`
+- `libxkbcommon`
+- `libinput`
+- `libgbm`
+
+If you want to enable X11 support (to run anvil in a X11 window for example),
+then you'll need to install the following packages as well:
+    - `xwayland`
```

Comment:
This is not exactly true, xwayland is used to run X11 clients inside of Anvil, aka. backwards compatibility with X11 clients.

Reply:
whoops, ok

Reply:
fixed